src/components/MusicList.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect, useCallback } from "react";
+import styled from "styled-components";
+import { useFetchTracksQuery } from "../features/spotify/spotifySlice";
+import Music from "./Music";
+import useDebounce from "../helpers/useDebounce";
+
+type MusicResultsProps = {
+  trackSearch: string;
+};
+
+const FlexContainer = styled.div`
+  justify-content: space-evenly;
+  display: grid;
+  grid-template-columns: 320px 320px 320px;
+`;
+
+const MusicList: React.FC<MusicResultsProps> = ({
+  trackSearch,
+}: MusicResultsProps) => {
```

Comment:
```suggestion
}) => {
```
Наличие `React.FC<MusicResultsProps>` уже должно было добавить тебе типы.